src/main/java/step2/view/ResultView.java

```diff
@@ -0,0 +1,36 @@
+package step2.view;
+
+import step2.domain.*;
+
+import java.util.List;
+
+public class ResultView {
+    private final static String PERSON_NAME_FORMAT = "%6s";
+    private final static String NEW_LINE = System.lineSeparator();
+
+    public static void print(Persons persons, Ladder ladder) {
+        arrangePersons(persons.getPersons());
+        System.out.print(NEW_LINE);
+        arrangeLadder(ladder);
+    }
+
+    private static void arrangePersons(List<Person> persons){
+       persons.stream()
+              .map(person -> formatName(person.getName()))
+              .forEach(name -> {
+                  System.out.print(name);
+              });
+    }
+
+    private static String formatName(String name) {
+        return String.format(PERSON_NAME_FORMAT, name);
+    }
+
+    private static void arrangeLadder(Ladder ladder) {
+        ladder.getLayers()
+              .stream()
+              .forEach(line -> {
+                  System.out.println(line.toString());
+              });
```

Comment:
forEach() 는 Iterable 인터페이스에 default 로 선언되어 있어 stream() 을 사용하지 않고도 사용이 가능해요.
그리고 System.out.println() 에 객체를 넘겨주게 되면 자동으로 toString() 을 호출하게 되어 별도로 toString() 을 호출해서 문자열을 넘겨주지 않아도 되어 아래처럼 사용해 볼 수 있어 보여요.
```suggestion
        ladder.getLayers()
              .forEach(System.out::println);
```